modules/ztp-preparing-the-ztp-git-repository.adoc

```diff
@@ -53,15 +53,23 @@ example
 │   ├── group-du-sno-ranGen.yaml
 │   ├── group-du-sno-validator-ranGen.yaml
 │   ├── kustomization.yaml
+│   ├── source-crs/
 │   └── ns.yaml
 └── siteconfig
     ├── example-sno.yaml
+    ├── reference-manifests/ <1> 
+    ├── custom-manifests/ <2>
     ├── KlusterletAddonConfigOverride.yaml
     └── kustomization.yaml
 ----
+<1> Contains reference manifests from the `ztp-container`.
+<2> Contains custom manifests.
 
 Keep `SiteConfig` and `PolicyGenTemplate` CRs in separate directories. Both the `SiteConfig` and `PolicyGenTemplate` directories must contain a `kustomization.yaml` file that explicitly includes the files in that directory.
 
 This directory structure and the `kustomization.yaml` files must be committed and pushed to your Git repository. The initial push to Git should include the `kustomization.yaml` files. The `SiteConfig` (`example-sno.yaml`) and `PolicyGenTemplate` (`common-ranGen.yaml`, `group-du-sno*.yaml`, and `example-sno-site.yaml`) files can be omitted and pushed at a later time as required when deploying a site.
 
-The `KlusterletAddonConfigOverride.yaml` file is only required if one or more `SiteConfig` CRs which make reference to it are committed and pushed to Git. See `example-sno.yaml` for an example of how this is used.
+In the `PolicyGentemplate` directory, create the `source-crs` directory and push the content found under `out/source-crs` to that directory.
+In the `SiteConfig` directory, create a subdirectory to hold the content found under `out/extra-manifests`, for example, `reference-manifests`.
+This directory should only contain CRs from the container, do not add any additional files. 
```

Comment:
Slightly confused by these two steps. Could we simply say: 
```suggestion
Copy the `out/extra-manifests` directory into the `SiteConfig` folder.
This directory should contain CRs from the `ztp-site-generate` container only. Do not add custom CRs here.  
```

Reply:
ok